Public/Src/Sandbox/MacOs/Interop/Interop.xcodeproj/xcshareddata/xcschemes/InteropLibrary.xcscheme

```diff
@@ -71,10 +71,10 @@
       </MacroExpansion>
    </ProfileAction>
    <AnalyzeAction
```

Comment:
>AnalyzeAction [](start = 4, length = 13)

Remove element? It seems empty...

Reply:
All of this is auto generated by the Xcode IDE depending on settings / schema changes. I'd rather not manually tweak that data. We had a small misconfiguration that got fixed, but thanks for the sanity checks 👍